expansion/src/main/java/no/unit/nva/expansion/model/ExpandedResource.java

```diff
@@ -31,22 +32,24 @@
 import java.util.stream.Stream;
 import java.util.stream.StreamSupport;
 import no.unit.nva.commons.json.JsonSerializable;
-import no.unit.nva.publication.external.services.UriRetriever;
 import no.unit.nva.identifiers.SortableIdentifier;
 import no.unit.nva.model.Publication;
+import no.unit.nva.publication.external.services.UriRetriever;
 import nva.commons.core.JacocoGenerated;
 import nva.commons.core.paths.UriWrapper;
 
 @SuppressWarnings("PMD.GodClass")
 @JsonTypeName(ExpandedResource.TYPE)
 public final class ExpandedResource implements JsonSerializable, ExpandedDataEntry {
-
     // The ExpandedResource differs from ExpandedDoiRequest and ExpandedMessage
     // because is does not extend the Resource or Publication class,
     // but it contains its data as an inner Json Node.
-    public static final String ID_FIELD_NAME = "id";
+
     public static final String TYPE = "Publication";
-    public static final String JSON_LD_CONTEXT_FIELD = "@context";
+    private static final String ID_FIELD_NAME = "id";
+    private static final String JSON_LD_CONTEXT_FIELD = "@context";
+    private static final String CONTEXT_TYPE_ANTHOLOGY = "Anthology";
+    private static final String INSTANCE_TYPE_ACADEMIC_CHAPTER = "AcademicChapter";
```

Comment:
It amuses me that we still use json-pointers unnecessarily — we should do something about that